src/libmongoc/doc/mongoc_client_encryption_opts_set_kms_providers.rst

```diff
@@ -18,23 +18,52 @@ Parameters
 * ``opts``: The :symbol:`mongoc_client_encryption_opts_t`
 * ``kms_providers``: A :symbol:`bson_t` containing configuration for an external Key Management Service (KMS).
 
-``kms_providers`` is a BSON document containing configuration for each KMS provider. Currently ``aws`` or ``local`` are supported. At least one must be specified.
+``kms_providers`` is a BSON document containing configuration for each KMS provider. Currently ``aws``, ``local``, ``azure``, ``gcp``, and ``kmip`` are supported. At least one must be specified.
 
 The format for "aws" is as follows:
 
 .. code-block:: javascript
 
    aws: {
-      accessKeyId: <string>,
-      secretAccessKey: <string>
+      accessKeyId: String,
+      secretAccessKey: String
    }
 
 The format for "local" is as follows:
 
 .. code-block:: javascript
 
    local: {
-      key: <96 byte BSON binary of subtype 0> // The master key used to encrypt/decrypt data keys.
+      key: byte[96] or String // The master key used to encrypt/decrypt data keys. May be passed as a base64 encoded string.
```

Comment:
C++-style comments for C code? It's fine if we're writing C99, but if our minimal support is C89, then perhaps we shouldn't...? 🤔